docs/contributing.md

```diff
@@ -0,0 +1,111 @@
+# Contributing to the JupyterHub Community
+
+JupyterHub is an open community made up of people all over the world.
+We invite participation of all forms - whether that be contributing
+code, advice, documentation, support, or just a friendly face!
+
+If you'd like to start getting involved with JupyterHub projects,
+here are a few resources to get you started.
+
+## Do I really have something to contribute to JupyterHub?
+
+Absolutely ✅. There are always ways to contribute to this community!
+
+If you have experience
+in some of the things that JupyterHub uses (e.g., shared infrastructure, dev-ops, kubernetes)
+then we would love your support in keeping these tools a useful resource for the community.
+
+If you don't have experience in these topics, then **contributing to JupyterHub is a great way to learn**.
```

Comment:
Let's turn this into a positive and talk about the many ways to contribute "before" we talk about how to contribute code. Similar to Mozilla's https://whatcanidoformozilla.org

Reply:
good point - I added a first paragraph to try and give people some pointers, what do you think?